pkg/apis/cincinnati/v1alpha1/cincinnati_types.go

```diff
@@ -39,6 +39,12 @@ type CincinnatiSpec struct {
 	// specified, it takes precedence over GitHubOrg and GitHubRepo and disables
 	// retrieval of graph data from GitHub.
 	GraphDataImage string `json:"graphDataImage,omitempty"`
+
+	// CofigmapKey refers to the ConfigMap key that holds a CA cert to the registry
+	// Cincinnati will contact to build the upgrade graph.  The name of the
+	// ConfigMap holding the key comes from the field additionalTrustedCA
+	// in the image.config.openshift.io API.
+	ConfigmapKey string `json:"configmapKey"`
```

Comment:
Make it `ConfigMapKey` with capital M? Or even `CertConfigMapKey` to give it some context.

Reply:
Ya I like CertConfigMapKey better than ConfigmapKey.